libopenage/engine.cpp

```diff
@@ -168,6 +168,19 @@ Engine::Engine(util::Dir *data_dir, const char *windowtitle)
 
 	glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
 
+	// qml sources will be installed to the asset dir
+	// otherwise assume that launched from the source dir
+	using namespace std::string_literals;
+	auto qml_search_paths = {this->data_dir->basedir, "libopenage/gui"s};
```

Comment:
isn't it automatically constructed? http://en.cppreference.com/w/cpp/string/basic_string/basic_string method (5)

(maybe not cause it's an rvalue)